public/data/saifrahman.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "Saif Rahman",
+    "type": "personal",
+    "bio": "Engineer-Student-Enthusiast",
+    "avatar": "https://avatars.githubusercontent.com/u/64410006?s=400&u=bb5179ef3d88c859be1907b535cbc6b1111372e6&v=4",
```

Comment:
```suggestion
    "avatar": "https://github.com/saifrahmn.png",
```

Reply:
@saifrahmn Please mark this change as resolved.